qpc/insights/login_config.py

```diff
@@ -1,13 +1,15 @@
 """AddLoginCommand is used to save insights login configuration."""
 
-from __future__ import print_function
+from logging import getLogger
 
 import qpc.insights as insights
 from qpc import messages
 from qpc.clicommand import CliCommand
 from qpc.translation import _
 from qpc.utils import write_insights_login_config
 
+LOG = getLogger('qpc')
```

Comment:
I suggest setting the logger to a lowercase variable. all uppercase is the convention for "constant" data.

Reply:
I fix this and pushed the final rebased version.